### PR TITLE
Obmin 377 create burger component

### PR DESCRIPTION
**Pull request by Viktoria-Serkiz**
Created burger component

### CODE REVIEW

**Review by Mish-hash — APPROVED** [review]

**Review by Voloshkina — APPROVED** [review]

**Review by Kyslytsia — APPROVED** [review]

File: src/components/burger/index.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+      {isOpen && (
+        <>
+          <Styles.BurgerMenu isAnimation={isAnimation}>
+            {burgerMenuText.map((item, index) => (
+              <React.Fragment key={index}>
+                <Deals text={item.text} to={item.to} heartIcon={item.icon} />
+              </React.Fragment>
+            ))}
+            <LanguageSelection lang="ua" onClick={() => {}} />
```
Comment (cloudmenson) [inline src/components/burger/index.tsx:27]: space between 26 and 27

File: src/components/burger/types.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export interface IBurger {
+  burgerMenuText: {
+    to: string;
+    text: string;
+    icon?: boolean;
+  }[];
+}
```
Comment (cloudmenson) [inline src/components/burger/types.ts:7]: Make typing on the example of Swiper, so clear and can be reused
Example:
export interface IMainSwiper {
  data: Data[];
}

export interface Data {
  src: string;
  href: string;
  title: string;
  subtitle: string;
}

File: src/components/burger/index.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { Deals, LanguageSelection, Responsive } from 'components';
+
```
Comment (Wolshebnik) [inline src/components/burger/index.tsx:4]: remove space

File: src/components/burger/index.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      {isOpen && (
+        <>
+          <Styles.BurgerMenu isAnimation={isAnimation}>
+            {burgerMenuItems.map((item, index) => (
+              <React.Fragment key={index}>
+                <Deals text={item.text} to={item.to} heartIcon={item.icon} />
+              </React.Fragment>
+            ))}
+
+            <LanguageSelection lang="ua" onClick={() => {}} />
```
Comment (Wolshebnik) [inline src/components/burger/index.tsx:28]: You need to run the callback from the outside

**Review by ASGladchenko — APPROVED** [review]

**Review by Wolshebnik — APPROVED** [review]

**Review by ASGladchenko — APPROVED** [review]

**Review by cloudmenson — APPROVED** [review]

**Pull request merged**